flink-table/flink-table-api-java/src/main/java/org/apache/flink/table/catalog/GenericInMemoryCatalog.java

```diff
@@ -48,6 +48,9 @@
  * A generic catalog implementation that holds all meta objects in memory.
  */
 public class GenericInMemoryCatalog implements Catalog {
+	public static final String FLINK_META_PROPERTY_KEY = "is_generic";
```

Comment:
seems duplicate to HiveCatalog.GENERIC_META_PROPERTY_KEY. 

We can either use 'is_generic' and `mask/retrieveFlinkProperties()` to deal with the `flink.` prefix, or just use 'flink.is_generic' after/before calls to `mask/retrieveFlinkProperties()`. Either way, we need to be consistent

Reply:
This is key is added automatically as part of metaobject instantiation. It's subject to mask/retrieve operations.